src/pages/ScoreReport.vue

```diff
@@ -839,6 +840,17 @@ const exportSelected = (selectedRows) => {
         ? _get(assessment, 'scores.computed.composite')
         : _get(assessment, `scores.computed.composite.${rawScoreKey}`);
       tableRow[`${taskDisplayNames[taskId]?.name ?? taskId} - Support Level`] = support_level;
+      if (assessment.reliable !== undefined && !assessment.reliable && assessment.engagementFlags !== undefined) {
```

Comment:
```suggestion
      if (!assessment.reliable && assessment.engagementFlags !== undefined) {
```
I believe that the first check may be redundant for our purposes since we don't care if `assessment.reliable` is null or undefined, just that it's falsy, but correct me if I am wrong.

Reply:
I think I prefer to keep them both for now, because those are 2 co-dependent things